lib/material/internal/material_menu.flow

```diff
@@ -2272,6 +2272,7 @@ MDropDownMenuStyle2MMenuStyle(style : [MDropDownMenuStyle]) -> [MMenuStyle] {
 
 MDropDownIcon(opened : Transform<bool>, style : [MDropDownIconStyle]) -> Material {
 	iconColorM = tryExtractStruct(style, MWhite());
+	iconOpacity = extractStruct(style, FillOpacity(0.54));
```

Comment:
I'd better use extractStructMany/tryExtractStruct in this case, so that default value declaration remains at MIcon2T.